src/execution_engines/test/security/node_js/load_test.spec.ts

```diff
@@ -0,0 +1,39 @@
+import NodeEngine from '../../../node_js'
+import { ExecutionEngines } from '../../../ExecutionEngine'
+
+describe('Puppeteer Load Test', () => {
+  beforeEach(() => {
+    process.env.ENGINE = ExecutionEngines.nodejs
+  })
+
+  // Once we have an actual JS based contract, we can update this spec to use
+  // it instead
+  it('measures the time to execute a trivial contract method 500 times', async () => {
```

Comment:
```suggestion
  it('Executing an simple array map 500 times...', async () => {
```
Good to be explicit here given the test is the one doing the timing